pkgs/applications/editors/imhex/default.nix

```diff
@@ -22,12 +22,15 @@
 }:
 
 let
-  version = "1.30.1";
+  # FIXME: https://github.com/NixOS/nixpkgs/pull/258964
+  # Currently unstable, because
```

Comment:
Self-referencing FIXME? Why is it unstable?

Reply:
Seems to be a mistake, is this ok?
```
  # FIXME: unstable, stable needs #252945 (details in #258964)
  # Next version bump should be stabilized
```

Reply:
Looks much better 😀